Sources/UtilityBeltData/CoreData/NSPersistentContainer+Mocked.swift

```diff
@@ -0,0 +1,31 @@
+// Copyright © 2020 SpotHero, Inc. All rights reserved.
+
+import CoreData
+
+public extension NSPersistentContainer {
+    /// Creates a mocked persistent container that stores data in memory instead of on disk.
+    /// - Parameter name: The name of the Data Model to persist.
+    static func mocked(name: String, storeType: StoreType = .memory) -> NSPersistentContainer {
+        let container = NSPersistentContainer(name: name) // , managedObjectModel: self.managedObjectModel)
```

Comment:
whats the commented code here? left over i assume?

Reply:
Yep, thanks!